tests/tests_create_raid_pool_then_remove.yml

```diff
@@ -0,0 +1,89 @@
+---
+- hosts: all
+  become: true
+  vars:
+    storage_safe_mode: false
+    storage_use_partitions: true
+    mount_location1: '/opt/test1'
+    mount_location2: '/opt/test2'
+    mount_location3: '/opt/test3'
+    volume1_size: '2g'
+    volume3_size: '3g'
+    volume2_size: '3g'
+
+  tasks:
+    - include_role:
+        name: storage
+
+    - include_tasks: get_unused_disk.yml
+      vars:
+        max_return: 2
+
```

Comment:
do we need set min_size here, 
if only one disk retrurned, then the min_size should be 8+ g,